dev/SplitButton/SplitButton.xaml

```diff
@@ -226,19 +237,23 @@
 
                         <Grid.ColumnDefinitions>
                             <ColumnDefinition x:Name="PrimaryButtonColumn" Width="*" MinWidth="{ThemeResource SplitButtonPrimaryButtonSize}"/>
-                            <ColumnDefinition x:Name="Separator" Width="1" />
```

Comment:
Discussed offline -- hopefully going to restore this and the divider can hang out there, if it works :)